core/sdk-core/src/main/java/software/amazon/awssdk/core/RequestOverrideConfiguration.java

```diff
@@ -148,6 +150,13 @@ public List<MetricPublisher> metricPublishers() {
         return metricPublishers;
     }
 
+    /**
+     * Return the plugins that will be used to update the configuration used by the request.
+     */
+    public List<SdkPlugin> registeredPlugins() {
+        return registeredPlugins;
+    }
```

Comment:
nit: just plugins() instead of registerdPlugins()? "registered" feels unnecessary.

Reply:
Sounds good, I will change it.

Reply:
Done